api/src/main/java/io/opentelemetry/OpenTelemetry.java

```diff
@@ -5,255 +5,194 @@
-import io.opentelemetry.internal.Obfuscated;
-import io.opentelemetry.metrics.DefaultMeterProvider;
 import io.opentelemetry.metrics.Meter;
 import io.opentelemetry.metrics.MeterProvider;
-import io.opentelemetry.metrics.spi.MeterProviderFactory;
-import io.opentelemetry.trace.DefaultTracerProvider;
 import io.opentelemetry.trace.Tracer;
 import io.opentelemetry.trace.TracerProvider;
-import io.opentelemetry.trace.propagation.HttpTraceContext;
-import io.opentelemetry.trace.spi.TracerProviderFactory;
-import java.util.Objects;
-import java.util.ServiceLoader;
-import javax.annotation.Nullable;
-import javax.annotation.concurrent.ThreadSafe;
 
 /**
- * This class provides a static global accessor for telemetry objects {@link Tracer}, {@link Meter}
- * and {@link BaggageManager}.
+ * The entrypoint to telemetry functionality for tracing, metrics and baggage.
  *
- * <p>The telemetry objects are lazy-loaded singletons resolved via {@link ServiceLoader} mechanism.
+ * <p>A global singleton can be retrieved by {@link #get()}. The default for the returned {@link
+ * OpenTelemetry}, will include any {@link io.opentelemetry.trace.spi.TracerProviderFactory}, {@link
+ * io.opentelemetry.metrics.spi.MeterProviderFactory}, or {@link
+ * io.opentelemetry.baggage.spi.BaggageManagerFactory} found on the classpath, or otherwise will be
+ * default, with no-op behavior.
  *
  * @see TracerProvider
- * @see MeterProviderFactory
- * @see BaggageManagerFactory
+ * @see MeterProvider
+ * @see BaggageManager
  */
-@ThreadSafe
-public final class OpenTelemetry {
-  private static final Object mutex = new Object();
-
-  @Nullable private static volatile OpenTelemetry instance;
-
-  private final TracerProvider tracerProvider;
-  private final MeterProvider meterProvider;
-  private final BaggageManager contextManager;
-
-  private volatile ContextPropagators propagators = DefaultContextPropagators.builder().build();
+public interface OpenTelemetry {
 
   /**
-   * Returns a singleton {@link TracerProvider}.
+   * Returns the registered global {@link OpenTelemetry}, a default {@link OpenTelemetry} composed
+   * of functionality any {@link io.opentelemetry.trace.spi.TracerProviderFactory}, {@link
+   * io.opentelemetry.metrics.spi.MeterProviderFactory}, or {@link
+   * io.opentelemetry.baggage.spi.BaggageManagerFactory} found on the classpath, or otherwise will
+   * be default, with no-op behavior.
    *
-   * @return registered TracerProvider or default via {@link DefaultTracerProvider#getInstance()}.
-   * @throws IllegalStateException if a specified TracerProvider (via system properties) could not
-   *     be found.
+   * @throws IllegalStateException if a provider has been specified by system property using the
+   *     interface FQCN but the specified provider cannot be found.
    */
-  public static TracerProvider getTracerProvider() {
-    return getInstance().tracerProvider;
+  static OpenTelemetry get() {
```

Comment:
Do we need to expose this on the public interface at this point? Could the callers here just make the call to `DefaultOpenTelemetry` directly?

Reply:
Oops, thanks for finding where I forgot to remove publicness, cleaned up